src/com/rcv/Tabulator.java

```diff
@@ -715,13 +715,32 @@ private boolean hasContinuingCandidates(SortedMap<Integer, Set<String>> rankToCa
       // this is used to determine how many skipped rankings occurred in the case of
       // undervotes
       int lastRank = 0;
-      // loop over all rankings in this CVR from most preferred to least and see how they will
-      // be rendered
+      // candidatesSeen is the set of candidates we've encountered while processing this CVR
+      // in this round; only relevant if exhaustOnDuplicateCandidate is enabled
+      Set<String> candidatesSeen = new HashSet<>();
+      // loop over all rankings in this CVR from most preferred to least until we reach a decision
```

Comment:
Is this comment related to the one below it? The wording is a bit confusing...

Reply:
I'll combine them so that it's clearer.